packages/google_maps_flutter/google_maps_flutter/example/integration_test/google_maps_test.dart

```diff
@@ -954,10 +955,8 @@ void main() {
     final BitmapDescriptor scaled = await BitmapDescriptor.fromAssetImage(
         imageConfiguration, 'red_square.png',
         mipmaps: false);
-    // ignore: invalid_use_of_visible_for_testing_member
-    expect(mip.toJson()[2], 1);
-    // ignore: invalid_use_of_visible_for_testing_member
-    expect(scaled.toJson()[2], 2);
+    expect((mip.toJson() as List<dynamic>)[2], 1);
```

Comment:
nit: assert size before reading the 2nd element

Reply:
While updating these I decided it wasn't worth adding complexity to the test code to assert incremental steps; if toJson returns a one-element array this test will fail with an out-of-bounds index exception, which gives us the same information than a length check would give us.

(The bigger problem is that this is testing code from the interface package; this test doesn't belong here at all. But moving it was out of scope for this change.)